src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  function getSum(a) {
+    return function(b) {
+      count++;
 
+      if (count < 4 || count % 2 !== 0) {
+        return a + b;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
you don't have to use else here - it won't ever reach that due to return in if statement above, please remove else